### ISSUE TITLE
receiptDate is incorrect for Walgreens receipts

### ISSUE THREAD

**Issue by chriskk** [BlinkReceipt/blinkreceipt-ios#24]
I've tried several receipts from April, but the receiptDate's value is May. 

For example:
* Receipt's actual date 04/13/2020, scanResults receiptDate is 05/13/2020
* Receipt's actual date 04/15/2020, scanResults receiptDate is 05/15/2020
* Receipt's actual date 04/17/2020, scanResults receiptDate is 05/17/2020

Tested on a different merchant's receipt, Molly Stone's in this case, and receipt date was correct.

**Comment by chriskk** [comment]
Tested on versions 1.8.6 and 1.8.9

**Comment by dpanzer** [comment]
Hi @chriskk - can you post photos of some of the receipts where the date is being returned incorrectly?

**Comment by chriskk** [comment]
![image](https://user-images.githubusercontent.com/486756/82490991-e96a8480-9a98-11ea-91a3-cb08a69e501f.jpeg)

**Comment by chriskk** [comment]
![image](https://user-images.githubusercontent.com/486756/82491322-59790a80-9a99-11ea-8d96-058f10bb9e12.jpeg)

**Comment by chriskk** [comment]
![image](https://user-images.githubusercontent.com/486756/82491075-043cf900-9a99-11ea-9bab-5673122ca523.jpeg)

**Comment by dpanzer** [comment]
Hi @chriskk - thanks for the images. I have reproduced the incorrect date on the 2nd and 3rd receipt, although I get the correct date on the first receipt (although I am scanning in a slightly different way than you - feeding these static images directly into our parsing engine). We will work on these cases and probably resolve by the next update. Thanks!

**Comment by chriskk** [comment]
Hi @dpanzer - Just wanted to follow up as we've noticed a drop in scans recently. Is this a BlinkReceipt server-side issue with Product Intelligence, or a client side SDK one? We've been using 1.8.6 for a while and never noticed the issue with dates. And we've had it out in our iOS app for over a month now.

Our scans have dropped down starting around May 20. We've also made some changes on our side, but wanted to get as much info as possible to narrow it down.
